sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_first_seen_v1/init.sql

```diff
@@ -0,0 +1,17 @@
+CREATE OR REPLACE TABLE
+  `firefox_accounts_derived.fxa_users_first_seen_v1`
+PARTITION BY
+  first_seen_date
+CLUSTER BY
+  user_id
+AS
+SELECT
+  user_id,
+  DATE(MIN(submission_timestamp)) AS first_seen_date,
+  ARRAY_AGG(DISTINCT service) AS services_used,
```

Comment:
```suggestion
  ARRAY_AGG(DISTINCT service ORDER BY service) AS services_used,
```

It's probably good to make the ordering here deterministic.